# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../luasocket/config-no-compat-5.1.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ruN luasocket-2.0-vanilla/config luasocket-2.0/config
--- luasocket-2.0-vanilla/config	2005-11-24 21:29:38.000000000 +0100
+++ luasocket-2.0/config	2006-03-14 05:19:14.000000000 +0100
@@ -12,39 +12,11 @@
 MIME_SO=mime.$(EXT).$(MIME_V)
 UNIX_SO=unix.$(EXT)
 
-#------
-# Lua includes and libraries
-#
-LUAINC=-Ilua-5.0.2/include
-LUALIB=-Llua-5.0.2/lib
-
-#------
-# Compat-5.1 directory
-#
-COMPAT=compat-5.1r4
-
-#------
-# Top of your Lua installation
-# Relative paths will be inside the src tree
-#
-INSTALL_TOP_SHARE=share
-INSTALL_TOP_LIB=lib
-
 INSTALL_DATA=cp
 INSTALL_EXEC=cp
 
 #------
 # Compiler and linker settings
-# for Mac OS X
-#
-CC=gcc
-DEF=-DLUASOCKET_DEBUG -DUNIX_HAS_SUN_LEN
-CFLAGS= $(LUAINC) -I$(COMPAT) $(DEF) -pedantic -Wall -O2 -fno-common
-LDFLAGS=-bundle -undefined dynamic_lookup
-LD=export MACOSX_DEPLOYMENT_TARGET="10.3"; gcc
-
-#------
-# Compiler and linker settings
 # for Linux
 #CC=gcc
 #DEF=-DLUASOCKET_DEBUG 
diff -ruN luasocket-2.0-vanilla/makefile luasocket-2.0/makefile
--- luasocket-2.0-vanilla/makefile	2005-11-24 21:29:38.000000000 +0100
+++ luasocket-2.0/makefile	2006-03-14 05:19:20.000000000 +0100
@@ -25,7 +25,6 @@
 	smtp.lua
 
 TO_TOP_SHARE:= \
-	$(COMPAT)/compat-5.1.lua \
 	ltn12.lua \
 	socket.lua \
 	mime.lua
diff -ruN luasocket-2.0-vanilla/src/makefile luasocket-2.0/src/makefile
--- luasocket-2.0-vanilla/src/makefile	2005-11-24 21:29:38.000000000 +0100
+++ luasocket-2.0/src/makefile	2006-03-14 05:20:24.000000000 +0100
@@ -11,7 +11,6 @@
 # Modules belonging to socket-core
 #
 SOCKET_OBJS:= \
-	$(COMPAT)/compat-5.1.o \
 	luasocket.o \
 	timeout.o \
 	buffer.o \
@@ -29,7 +28,6 @@
 # Modules belonging mime-core
 #
 MIME_OBJS:=\
-	$(COMPAT)/compat-5.1.o \
 	mime.o 
 
 
